embassy-boot-rp/Cargo.toml

```diff
@@ -46,6 +46,8 @@ log = [
     "embassy-boot/log",
     "embassy-rp/log",
 ]
+# Feature available for semver checking. In general: enable directly on embassy-rp
```

Comment:
oh wow this is unfortunate. Do you think it's possible to fix this in cargo-semver-checks?